TakeOne/Assets/_Project/Scripts/Dice/DiceDragger.cs

```diff
@@ -107,8 +107,13 @@ private void SetCurrentDice()
             var ray = diceCam.ScreenPointToRay(Input.mousePosition);
             if (Physics.Raycast(ray, out var hit))
             {
-                if (!hit.collider.gameObject.CompareTag("Dice")) return;
-                _currentDice = hit.collider.gameObject.GetComponent<DiceController>();
+                if (!hit.collider.CompareTag("Dice")) return;
+                _currentDice = hit.collider.GetComponent<DiceController>();
```

Comment:
Shouldn't we only assign the dice if it's both selectable and tagged dice?
Wouldn't nulling it after the fact mean if we select a dice, then click a relic dice we would no longer have the first dice selected?
is that an intended functionality?